scipy/optimize/nonlin.py

```diff
@@ -868,7 +868,7 @@ def svd_reduce(self, max_rank, to_retain=None):
         D, R = qr(D, mode='economic')
         C = dot(C, R.T.conj())
 
-        U, S, WH = svd(C, full_matrices=False, compute_uv=True)
```

Comment:
Having explicit keywords are encouraged so we don't need to change it here.

Reply:
I think this is an okay cleanup, giving the default value of some keyword is more confusing that helpful.